### ISSUE TITLE
Error in training resolved

### ISSUE THREAD

**Issue by ushagayatri** [jiegzhan/multi-class-text-classification-cnn#5]
To switch to tensoeflow 0.9 on python 3-

export TF_BINARY_URL=https://storage.googleapis.com/tensorflow/mac/tensorflow-0.9.0rc0-py3-none-any.whl
python3 -m pip install $TF_BINARY_URL

Then at line 82
# losses = tf.nn.softmax_cross_entropy_with_logits(self.scores, self.input_y)

losses = tf.nn.softmax_cross_entropy_with_logits(labels = self.input_y, logits = self.scores) #  only named arguments accepted

This worked for me! @jiegzhan , Do you agree with this?
